upb/def.c

```diff
@@ -667,6 +686,11 @@ const upb_msglayout *upb_msgdef_layout(const upb_msgdef *m) {
   return m->layout;
 }
 
+const upb_extrange *upb_msgdef_extrange(const upb_msgdef *m, int i) {
+  UPB_ASSERT(i >= 0 && i < m->ext_range_count);
```

Comment:
0 <= i && i < size

reads more cleanly for me

Reply:
I'm happy to move towards that style. For consistency with the many other instances in this file (like the next function below) I'd prefer to keep this as written for now.